docs/sources/shared/intro/profile-types-descriptions.md

```diff
@@ -0,0 +1,75 @@
+---
+headless: true
+description: Shared file for profile types.
+---
+
+[//]: # 'Profile types descriptions.'
+[//]: # 'This shared file is included in these locations:'
+[//]: # '/pyroscope/docs/sources/introduction/profiling-types.md'
+[//]: # '/website/content/grafana-cloud/monitor-applications/profiles/introduction/profiling-types.md'
+[//]: #
+[//]: # 'If you make changes to this file, verify that the meaning and content are not changed in any place where the file is included.'
+[//]: # 'Any links should be fully qualified and not relative: /docs/grafana/ instead of ../grafana/.'
+
+<!-- Profile type descriptions -->
+
+## CPU profiling
+
+CPU profiling measures the amount of CPU time consumed by different parts of your application code.
+High CPU usage can indicate inefficient code, leading to poor performance and increased operational costs.
+It's used to identify and optimize CPU-intensive functions in your application.
```

Comment:
You do such a fantastic job of introducing concepts throughout all of these topics, Kim!